packages/framework/package.json

```diff
@@ -2,8 +2,10 @@
   "name": "@novu/framework",
   "version": "2.3.0",
   "description": "The Code-First Notifications Workflow SDK.",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "main": "./dist/cjs/index.cjs",
+  "types": "./dist/cjs/index.d.cts",
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** Needs to use the `.cjs` extension for top-level `types` property